modules/apps/portal/portal-tika/src/main/java/com/liferay/portal/tika/internal/extract/TextExtractorImpl.java

```diff
@@ -227,9 +229,12 @@ public String call() throws ProcessException {
 
 			logger.setLevel(Level.SEVERE);
 
-			Tika tika = new Tika(TikaConfigUtil.getTikaConfig());
-
 			try {
+				Tika tika = new Tika(
+					new TikaConfig(
```

Comment:
@jorgediaz-lr I do not understand why you are doing this. What kind of issue you would have when you reading the configuration in a child process?